Add Tor process keep running after closing Wasabi FAQ (#943)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-GeneralBitcoinPrivacy.md

## The Privacy of Tor

:::details
### Why does Tor process keep running after closing Wasabi Wallet?

There are two main reasons: user experience and privacy.

> If an user closes Wasabi and then starts it again, he's already connected to Tor and he does not need to build a new Tor circuit.

Reviewer: ```suggestion

If n user closes Wasabi and then starts it again, he's already connected to Tor and he does not need to build a new Tor circuit.
```

> That means that the Wasabi startup will be much faster, because the Tor circuit is already estabilished.

Reviewer: ```suggestion
This means that the next Wasabi startup will be much faster, because the Tor circuit is already established.
```

> Regarding the privacy, this is done for avoiding time correlations attacks when opening Wasabi Wallet.

Reviewer: ```suggestion
Regarding the privacy, this is done to avoid time correlation attacks when using Wasabi Wallet.
```

> In fact, if a user were to create a new Tor circuit every time he opens Wasabi Wallet, he would send a series of information to the bitcoin network, which could allow a correlation and deanonymization of the user himself.
> 

Reviewer: ```suggestion
In fact, if a user creates a new Tor circuit every time he opens Wasabi Wallet, he would send a series of information to the Bitcoin network, which could allow a correlation and deanonymization of the user himself.
```

You can also configure many cloud storage providers in this way.
:::

:::details
### Why does Tor process keep running after closing Wasabi Wallet?

There are two main reasons: user experience and privacy.

If a user closes Wasabi and then starts it again, he's already connected to Tor and he does not need to build a new Tor circuit.

> This means that the next Wasabi startup will be much faster, because the Tor circuit is already established.

Reviewer: ```suggestion
This means that Wasabi's startup will be much faster, because the Tor circuit is already established.
```
